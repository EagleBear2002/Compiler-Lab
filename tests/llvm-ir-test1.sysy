int main(){
    return (5-(2+3))*2/1;
}
