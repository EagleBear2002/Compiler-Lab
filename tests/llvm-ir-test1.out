; ModuleID = 'moudle'
source_filename = "moudle"

define i32 @main() {
mainEntry:
  ret i32 -2
}
